src/types/to_sql.rs

```diff
@@ -87,37 +87,27 @@ pub trait ToSql {
     fn to_sql(&self) -> Result<ToSqlOutput<'_>>;
 }
 
-impl ToSql for Box<dyn ToSql> {
```

Comment:
Does this still work covered on the other implementation? If not I don't know if we can take this -- it's useful and common.

Reply:
I think yes, because `?Sized` should work for unsized (dyn types) that implement `ToSql`